phoenix-core/src/main/java/org/apache/phoenix/parse/CreateTableStatement.java

```diff
@@ -83,11 +87,13 @@ public CreateTableStatement(CreateTableStatement createTable, ListMultimap<Strin
         this.baseTableName = createTable.baseTableName;
         this.whereClause = createTable.whereClause;
         this.immutableRows = createTable.immutableRows;
+        this.familyCQCounters = createTable.familyCQCounters;
     }
 
     protected CreateTableStatement(TableName tableName, ListMultimap<String,Pair<String,Object>> props, List<ColumnDef> columns, PrimaryKeyConstraint pkConstraint,
-            List<ParseNode> splitNodes, PTableType tableType, boolean ifNotExists, 
-            TableName baseTableName, ParseNode whereClause, int bindCount, Boolean immutableRows) {
+                                   List<ParseNode> splitNodes, PTableType tableType, boolean ifNotExists,
+                                   TableName baseTableName, ParseNode whereClause, int bindCount, Boolean immutableRows,
+                                   Map<String, Integer> familyCounters) {
```

Comment:
`ColumnDef` class stores family names as `NamedNode`, which is a `String` and a `boolean` flag for case sensitivity. So using String here was a problem, but not because hbase, rather because the case sensitive cases. 

Family names for the counters are now stored as `NamedNode`, also few new tests have been added.

Thank you for pointing this out.